lib/const.rb

```diff
@@ -50,31 +50,31 @@
 # Glibc version can be found from the output of libc.so.6
 LIBC_VERSION = `/#{ARCH_LIB}/libc.so.6`[/Gentoo ([^-]+)/, 1]
 
-if ENV['CREW_PREFIX'].to_s.empty? || (ENV['CREW_PREFIX'] == '/usr/local')
-  CREW_BUILD_FROM_SOURCE = ENV.fetch('CREW_BUILD_FROM_SOURCE', nil)
-  CREW_PREFIX = '/usr/local'
+CREW_PREFIX = ENV.fetch('CREW_PREFIX', '/usr/local')
+
+if CREW_PREFIX == '/usr/local'
+  CREW_BUILD_FROM_SOURCE = !!ENV['CREW_BUILD_FROM_SOURCE']&.eql?('1')
```

Comment:
What is wrong with `ENV.fetch` or simply `CREW_BUILD_FROM_SOURCE = ENV['CREW_BUILD_FROM_SOURCE']`?

Reply:
Some people might be uncomfortable with actually unsetting a variable with `unset` (like me :) ), they would probably run something like `VAR=''` to "unset" them (the env variable still exists but being empty).

Also, this prevents unexpected results and misunderstanding, like `export CREW_BUILD_FROM_SOURCE=false` being evaluated as `true`